PatientAnalytics/Models/User.cs

```diff
@@ -1,10 +1,12 @@
 using System.ComponentModel.DataAnnotations;
 using System.Text.Json.Serialization;
+using Microsoft.EntityFrameworkCore;
 using PatientAnalytics.Middleware;
 using PatientAnalytics.Models.Auth;
 
 namespace PatientAnalytics.Models;
 
+[Index(nameof(Username), nameof(Email), IsUnique = true)]
```

Comment:
Is it worth doing this on your db context rather than an attribute? I’d suggest being consistent with how you do it - either all attributes or all fluent interface - makes it easier to find things

Reply:
Ah yes, makes sense. I will do it on my db context.